.github/workflows/pull-request-conditionals.yaml

```
@@ -85,3 +85,14 @@ jobs:
       flavor: ${{ matrix.flavor }}
       test_type: ${{ matrix.test_type }}
     secrets: inherit # Inherits all secrets from the parent workflow.
+
+  evaluate-package-compliance:
+    needs: run-package-test
+    name: Compliance Evaluation
+    strategy:
+      matrix:
+        flavor: [upstream, registry1, unicorn]
```

```suggestion
    strategy:
      matrix:
        flavor: [upstream, registry1, unicorn]
      fail-fast: false
```

Suggestion - I think this makes sense? Would want all compliance evaluated each time vs cancelling parallel flavors on one failing (see https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#handling-failures).

---

Given that the preceding job `run-package-test` is required for this job to execute - the runtime of this job means that everything to this point has passed yes? 

If so, given the overhead of `evaluate` which is merely static file processing (seconds at most) - I could see value in always having all flavors evaluated and available for comparison. One failing might signify a problem with a specific flavor vs all failing might be more of a systemic issue.

Open to thoughts

---

Yeah I agree with all of the above - which I think means you're aligned with my suggestion? 😅 

As-is `fail-fast` is `true` by default. Specifically for this suggestion I think this means that all evaluations run in parallel but if one fails the others will be cancelled before they complete. In practice I'm not sure I've seen that (ex: [this one](https://github.com/defenseunicorns/uds-core/actions/runs/9942641965) seemed to just run both to completion) - possibly because the jobs take almost identical time to run and there is no time to cancel the others?

What I wouldn't want is a situation where the unicorn flavor evaluation fails and causes the registry1 evaluation to be cancelled - leading to no comparison of results. Would always be beneficial to have all results, as you noted if all flavors are failing there may be a systemic issue vs flavor specific.

---

💯 I have watched these pipelines execute many times and never observed that cancel happening -> but it could certainly be latency and GitHub unable to cancel them fast enough. Agree with the suggestion.